src/v2/Components/DownloadAppBadge.tsx

```diff
@@ -83,3 +95,11 @@ export const DownloadAppBadge: React.FC<DownloadAppBadgeProps> = track(null, {
     </Link>
   )
 })
+
+const tracks = {
+  clickedAppDownload: (options: AnalyticsOptions): ClickedAppDownload => ({
+    action: ActionType.clickedAppDownload,
+    subject: "Download on the App Store",
+    ...options,
+  }),
+}
```

Comment:
I'm tempted to say we actually don't need this `tracks` object in this case. It's a pattern we have in other places, but here, it feels like it just leads to misdirection. We're assembling an object out of two separate objects that don't really need to be separate, and that means we have to do some extra typing and type declaring.

Instead of what we have here, we could just do:

```ts
 const clickedAppDownload: ClickedAppDownload = {
      action: ActionType.clickedAppDownload,
      context_module: contextModule,
      context_page_owner_type: contextPageOwnerType!,
      context_page_owner_slug: contextPageOwnerSlug,
      context_page_owner_id: contextPageOwnerId,
      destination_path: downloadAppUrl,
      subject: "Download on the App Store",
    }

    tracking.trackEvent(clickedAppDownload)
```

That said, standards are useful to have, so I'm not sure what the right call is. What do you think @lidimayra?

Reply:
I think this is much better!!!
I just followed the standard because I'm not that comfortable when working in the front-end, so I just copied what was done in other places... but I do think this is much simpler. I just applied the changes here and now it looks much easier to understand what's going on.

So I squashed this into 2ad327d95cc1387abb395b50200d204663b4177a (and applied similar changes to `SellWithArtsy` component and squashed it into 725bc1cda4a59e2b643dc0787bae079bac634d65)